packages/core/src/common/classes.ts

```diff
@@ -7,207 +7,239 @@
 import { Alignment } from "./alignment";
 import { Intent } from "./intent";
 
+const NS = "bp3";
```

Comment:
this namespace is bp3 but the one above is pt?

Reply:
yeah oops shhhhhh